src/scss/components/_dropdown.scss

```
@@ -0,0 +1,44 @@
+@use '../theme' as *;
+@use '../utilities/mixins' as *;
+
+.dropdown {
+  padding: var(--spacer-x-small);
+  min-width: 331px;
+  text-align: right;
+  display: inline-block;
+
+  button {
+    font-size: $font-size-label;
+    font-weight: var(--font-weight-light);
+    text-decoration: none;
+    padding: 0.1875rem;
+
+    &.chevron::after {
```

Do we need a separate `.chevron` class? I think instead we should just relate this `::after`  to `[aria-expanded]` keeping it simple.
```suggestion
    &[aria-expanded]::after {
```